src/js/chapter-factory.js

```diff
@@ -0,0 +1,87 @@
+import { makeDraggable, makeDroppable } from "./drag-drop";
+
+//Chapter factory
+const Chapter = (story, images, question, chapterNumber) => {
+  let completed = false;
+
+  const changeStatus = () => (completed = !completed);
+  const getStatus = () => completed;
+
+  return { story, images, question, chapterNumber, changeStatus, getStatus };
+};
+
+//Chapter DOM structure
+const createChapterStructure = (chapterObj) => {
+  const createButtons = () => {
+    const homeButton = document.createElement("button");
+    const nextChapterButton = document.createElement("button");
+
+    homeButton.textContent = "home";
+    homeButton.classList.add("home");
+    nextChapterButton.textContent = "-->";
+    nextChapterButton.classList.add("next");
+
+    return { homeButton, nextChapterButton };
+  };
+
+  const setupImages = () => {
+    const images = [...chapterObj.images];
+    const gallery = document.createElement("div");
+
+    for (let image of images) {
+      const picture = document.createElement("img");
+
+      picture.src = image.url;
+      picture.classList.add(image.cssClass);
+      picture.style.height = "100px";
```

Comment:
Put this styling into css and just add a class here e.g. 'drag-image'.

Reply:
Yeah, i agree, i just used them to quickly change images size to test if drag and drop works correctly.